docs/users-guide/interactive-dashboards.md

```diff
@@ -0,0 +1,104 @@
+## Interactive dashboards
+
+You can customize what happens when people click on questions in your dashboard.
+
+By default, when you create charts using Metabase's graphical query builder, your charts automatically come with [drill-through capabilities](https://www.metabase.com/blog/drilling-through-data/index.html), which let folks click on a chart to explore further. But if you have a more customized click path in mind, Metabase allows you to customize what happens when a user clicks on a chart or table in your dashboard.
+
+You can set up a dashboard card to:
+
+- Send the user to a custom destination: a dashboard, question, or custom URL.
+- Update a dashboard filter (see [use a chart to filter a dashboard](#Use-a-chart-to-filter-a-dashboard)).
+
+To configure this interactivity, you'll use the **click behavior** option on a dashboard card to slide out the click behavior.
+
+### Customizing click behavior
+
+From your dashboard, click on the **pencil** icon to enter dashboard edit mode. Hover over the card containing the question you want to customize. Metabase will display a menu at the top right of the card. Select the **Click behavior** option (the icon with the mouse cursor clicking on a card).
+
+![Click behavior icon](images/interactive-dashboards/click-behavior-icon.png)
+
+Metabase will slide out the **Click behavior sidebar**. For questions composed using the query builder, you can select from 3 options:
```

Comment:
Stylistc preference, but I want to say the Chicago Manual of Style recommends spelling out numbers that are less than 10.

```suggestion
Metabase will slide out the **Click behavior sidebar**. For questions composed using the query builder, you can select from three options:
```